packages/beacon-node/package.json

```diff
@@ -129,7 +129,7 @@
     "@chainsafe/pubkey-index-map": "^3.0.0",
     "@chainsafe/ssz": "^1.2.2",
     "@chainsafe/threads": "^1.11.3",
-    "@crate-crypto/node-eth-kzg": "0.8.0",
+    "@crate-crypto/node-eth-kzg": "0.9.1",
```

Comment:
curious about 0.10.0 which includes https://github.com/crate-crypto/rust-eth-kzg/pull/433, don't we want it to be multithreaded?